src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,87 @@
-export const Pagination = () => {};
+import React from 'react';
+import { getNumbers } from '../../utils';
+import cn from 'classnames';
+
+type Props = {
+  totalItems: string[];
+  itemsPerPage: number;
+  currentPage: number;
+  onPageChange: (value: number) => void;
+  startItem: number;
+  endItem: number;
+};
+
+export const Pagination: React.FC<Props> = ({
+  totalItems,
+  itemsPerPage,
+  currentPage,
+  onPageChange,
+  startItem,
+  endItem,
+}) => {
+  const numberOfPages: number = Math.ceil(totalItems.length / itemsPerPage);
+  const pages = getNumbers(1, numberOfPages);
+
+  return (
+    <>
+      <ul className="pagination">
+        <li
+          className={cn('page-item', {
+            disabled: currentPage === pages[0],
+          })}
+        >
+          <a
+            data-cy="prevLink"
+            className="page-link"
+            href="#prev"
+            aria-disabled={currentPage === pages[0]}
+            onClick={() =>currentPage !== pages[0] && onPageChange(currentPage - 1)}
```

Comment:
The `onClick` handler should prevent the default action of the anchor tag to avoid potential issues with the URL fragment changing. You can do this by adding `event.preventDefault()` at the beginning of the handler.